frontend/src/data/parseReportData.ts

```diff
@@ -118,8 +196,9 @@ export interface ImplementationMetadata {
   name: string;
   version?: string;
   dialects: string[];
-  homepage?: string;
+  homepage: string;
```

Comment:
Homepage is left optional in the current schema:

https://github.com/bowtie-json-schema/bowtie/blob/main/bowtie/schemas/io-schema.json#L70

Probably it shouldn't be, so feel free to constrain it there, but be careful when making changes that they match the schema.

Reply:
I believe we only need to include `homepage` in the **required** array. Is there anything else we should modify to make `homepage` a required field?

Reply:
> I believe we only need to include homepage in the required array.

That seems right yep.

Reply:
This is still outstanding right?